### DISCUSSION TITLE
Commit Not Saving To SQL File After Editing Data

### DISCUSSION THREAD

**Discussion by russtanner** [pawelsalawa/sqlitestudio#3927] [Q&A]
SQLiteStudio 3.2.1 on Win 10 Pro version 2004 Build 19041.572

When using sqlitestudio, I add a row to a table and commit the data (CTRL-Enter or Click the commit button). 

When I close and re-open the table, the new row is gone. It appears it's not being saved to the file.

I have been searching for quite a while and cannot find a solution.
Any help would be greatly appreciated.

**Comment by russtanner** [comment]
I found the cause of the issue.
The database file is named "databaseÆ.db". Note the special character (Æ).
When I renamed the database "database-active.db", it worked as expected including committing edits properly.

As far as I can tell, it appears that Studio cannot handle special characters in file names.
**Is this something that can be added to a request (or bug) list?**
Thanks.

**Reply by pawelsalawa** [reply]
Have you tried the official "sqlite3.exe" tool from sqlite.org? Does it work properly with this special character?
